gems/pending/appliance_console/utilities.rb

```diff
@@ -3,9 +3,26 @@
 require "appliance_console/internal_database_configuration"
 require "util/postgres_admin"
 require "awesome_spawn"
+require "rake"
+require "English"
 
 module ApplianceConsole
   module Utilities
+    def self.rake(task, *args)
+      rake_pid = Process.fork do
+        $stderr.reopen(LOGFILE, "a")
+        ARGV.push(*args)
+        Dir.chdir(RAILS_ROOT) do
+          app = Rake.application
+          app.init
+          app.load_rakefile
+          app[task].invoke
+        end
+      end
+      Process.wait(rake_pid)
+      $CHILD_STATUS.exitstatus
```

Comment:
If we are not reading the output, do we want to have awesome spawn write directly to the log file?

hmm, guess we would use something other than capture3 for that.